### PR TITLE
Jupyter Support + Bug Fix

### LINKED ISSUE #7: Ability to use AssertMeasurement on multiple qubits
Currently its hardcoded to one.

### PR DESCRIPTION
**Pull request by crazy4pi314**
So this fixes #7 and in the process adds the remaining features needed to use this simulator from Jupyter Notebook (see sample notebook in the samples folder)

### TODO:
- [x] Get package building and hosted from GH
~- [ ] Add feed to project repo base so ppl who don't build locally can use in Jupyter~

### CODE REVIEW

**Comment by crazy4pi314** [comment]
Ok I remember now why I can't have a config that adds the GH package registry feed, I'll remove and remove requirements for now.

File: src/jupyter/ChpMagic.cs [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        private IConfigurationSource configurationSource;
+
+        /// <summary>
+        /// Default constructor.
+        /// </summary>
+        public ChpMagic(ISymbolResolver resolver, IConfigurationSource configurationSource) : base(
+            "chp",
+            new Documentation
+            {
+                Summary = "Runs a given function or operation on the ToffoliSimulator target machine.",
```
Comment (RolfHuisman) [inline src/jupyter/ChpMagic.cs]: think this is the CHP simulator

File: src/StabilizerSimulator.cs [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
 {
     public class StabilizerSimulator : QuantumProcessorDispatcher
     {
-        public StabilizerSimulator() : base(new StabilizerProcessor())
+        public StabilizerSimulator(int? nQubits = null) : base(new StabilizerProcessor(nQubits))
         {
             (this.QuantumProcessor as StabilizerProcessor).Simulator = this;
         }
+
+        internal new void MaybeDisplayDiagnostic(object displayable) =>
```
Comment (RolfHuisman) [inline src/StabilizerSimulator.cs:32]: Maybe diagnostic ? Are we not sure we want diagnostics  :) ?

**Review by RolfHuisman — CHANGES_REQUESTED** [review]
We should clean up the descriptions.

**Review by RolfHuisman — APPROVED** [review]

**Review by RolfHuisman — APPROVED** [review]

**Pull request merged**